index.html

```diff
@@ -35,79 +49,197 @@
+                <div class="div-4">
+                    <h2 class="header-4">Profesional Art Printing Data</h2>
+                    <p class="para-3">A daily selection of privately personalized 
+                        reads; no accounts or sign-ups required. 
+                        Has been the industry's standard.</p>
+                </div>
+                <div class="tech-stack-2">
+                    <h4 class="h4-5">HTML</h4>
+                    <h4 class="h4-6">Bootstrap</h4>
+                    <h4 class="h4-7">Ruby</h4>
+                </div>
+                <button class="button-2 button-2-desk" type="submit">See Projects</button>
+            </section>
+
+            <!-- my-works 5/6-->
+            
+            <section class="sec section-3">
+                <img src="images/project-example.png" alt="work example" class="img-w1">
+                <div class="div-4">
+                    <h2 class="header-4">Profesional Art Printing Data</h2>
+                    <p class="para-3">A daily selection of privately personalized 
+                        reads; no accounts or sign-ups required. 
+                        Has been the industry's standard.</p>
+                </div>
+                <div class="tech-stack-2">
+                    <h4 class="h4-5">HTML</h4>
+                    <h4 class="h4-6">Bootstrap</h4>
+                    <h4 class="h4-7">Ruby</h4>
+                </div>
+                <button class="button-2 button-2-desk" type="submit">See Projects</button>
+            </section>
+            
+            <!-- my-works 6/6-->
+            
+            <section class="sec section-3">
+                <img src="images/project-example.png" alt="work example" class="img-w1">
+                <div class="div-4">
+                    <h2 class="header-4">Profesional Art Printing Data</h2>
+                    <p class="para-3">A daily selection of privately personalized 
+                        reads; no accounts or sign-ups required. 
+                        Has been the industry's standard.</p>
+                </div>
+                <div class="tech-stack-2">
+                    <h4 class="h4-5">HTML</h4>
+                    <h4 class="h4-6">Bootstrap</h4>
+                    <h4 class="h4-7">Ruby</h4>
+                </div>
+                <div>
+                    <button class="button-2 button-2-desk" type="submit">See Projects</button>
+                </div>
+            </section>
+        </div>
+
+        <!-- my-works 1/6 for desktop -->
+
+        <div class="grid-3-desktop">
+            <section class="sec section-3-desktop">     
+                <img src="images/img-placeholder.png" alt="work example" class="img-w1">
+                <button class="button-2" type="submit">See Projects</button>
+            </section>
+
         <!-- my-works 2/6-->
+
         <section class="sec section-3">
```

Comment:
- [ ] [OPTIONAL] The image on the left side indicates the state of the cards when hovering over them. All cards should look like the one on the right and change to the one on the left when hovering over them. It would be great if you implement it as such. 😎️

![image](https://user-images.githubusercontent.com/17243325/236600654-04411b3f-08ea-41db-81e0-c4394dd73aaf.png)

Reply:
@amejid Hi!! Thank you. That's amazin. I will try to do it and see how it's work. Thank you so much